src/App.tsx

```diff
@@ -1,14 +1,39 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { SelectCategory } from './types/SelectCategory';
+import { Todo } from './types/Todo';
+import { filterTodos } from './components/utils/filterTodos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [query, setQuery] = useState('');
+  const [selectTodoId, setSelectTodoId] = useState<number | null>(null);
+  const [selectCaregory, setSelectCategory] = useState<SelectCategory>(
+    SelectCategory.all,
+  );
+
+  const selectedTodo = todos.find(todo => todo.id === selectTodoId);
+
+  const filtredTodos = filterTodos(todos, { selectCaregory, query });
```

Comment:
The variable `filtredTodos` contains a typo. It should be `filteredTodos` to correctly represent the filtered list of todos.